pkg/credentials/helper.go

```diff
@@ -55,7 +57,32 @@ func (h *HelperStore) GetAll() (map[string]types.AuthConfig, error) {
 		return nil, err
 	}
 
-	for serverAddress := range serverAddresses {
+	newCredAddresses := make(map[string]string, len(serverAddresses))
+	for serverAddress, val := range serverAddresses {
```

Comment:
Interesting... that's weird behavior from mac library

Reply:
Yep it definitely took me a bit to figure that one out lol